src/server/routes/categoryRoutes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { getAllCategories, createCategory } from '../controllers/categoryController.js';
+
+const router = express.Router();
```

Comment:
change name for each routr that you do to him export. 
cant be the same name for many vars